### DISCUSSION TITLE
Galaxy Buds + (2021)

### DISCUSSION THREAD

**Discussion by shadowwulf** [timschneeb/GalaxyBudsClient#255] [General]
This program is working as advertised, much better  than the Galaxy Buds progream from MS store that only supports Galaxy buds Pro.  Hope this continues as a project. Thanks to the author for me finding this. Literally almost the same thing as the Android app.

**Comment by shadowwulf** [comment]
i do have a question tho, do these headsets not understand multiple game inputs ? seems at one point i could play a game and listen to music and all was fine until recently.

**Reply by NormalHuman-Anything** [reply]
if your game uses microphone, in my case, it seems like if its input is set to (your galaxy buds model name) hands-free headset(somewhat similar), you have to set your ouptput with that same named output device.
it worked for me..
